metadata-integration/java/datahub-protobuf/README.md

```diff
@@ -438,6 +493,34 @@ Set the domain id for the dataset. The domain should exist already. Note that th
   }
 ```
 
+##### DEPRECATION
+
+Deprecation of fields and messages are natively supported by protobuf options, however field level deprecation is limited to adding a tag in DataHub.
```

Comment:
Maybe note that we add a specific tag with URN / Name here. 

Are we also supporting deprecation at the parent dataset level?

Reply:
It seems like yes - consider also noting that for Event-level deprecation we use the standard "Deprecation" aspect